src/scss/objects/_buttons.scss

```diff
@@ -54,48 +64,52 @@
     * 3. Make buttons inherit font styles.
     * 4. Force all elements using beautons to appear clickable.
     * 5. Normalise box model styles.
-    * 6. If the button’s text is 1em, and the button is (3 * font-size) tall, then
-    *    there is 1em of space above and below that text. We therefore apply 1em
-    *    of space to the left and right, as padding, to keep consistent spacing.
-    * 7. Fixes odd inner spacing in IE7.
+    * 6. Fixes odd inner spacing in IE7.
```

Comment:
We can probably delete this line, if it's for an IE7 fix?

Reply:
Yeap, tested and removed